### PR TITLE
Pd 207 add info about metadata disk

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

**Comment by andreipope** [comment]
Added an example of how to run `px-runc` with the `-metadata` flag and made a bunch of edits. I think it reads a bit better now.

File: content/install-with-other/docker/shared/runc-install-bundle.md [hunk 1]
```diff
@@ -23,4 +21,4 @@ sudo docker run --entrypoint /runc-entry-point.sh \
     $latest_stable
 ```
 
-{{<info>}}Running the PX OCI bundle does not require Docker, but Docker will still be required to _install_ the PX OCI bundle.  If you do not have Docker installed on your target hosts, you can download this Docker package and extract it to a root tar ball and manually install the OCI bundle.{{</info>}}
+{{<info>}}Running the Portworx OCI bundle does not require Docker, but **Docker is required to install the Portworx OCI bundle**.  If you do not have Docker installed on your target hosts, you can download this Docker package and extract it to a root tarball. Next, manually install the OCI bundle.{{</info>}}
```
Comment (nathan-px) [inline content/install-with-other/docker/shared/runc-install-bundle.md]: This wording seems awkward to me. Maybe something like this is clearer:

```suggestion
{{<info>}}Your target hosts do not require Docker to run the Portworx OCI bundle, but they do require Docker to install it. If you have not installed Docker on your target hosts, you can install the OCI bundle by doing the following:

1. Download this Docker package and extract it to a root tarball
2. Manually install the OCI bundle{{</info>}}
```

File: content/install-with-other/docker/standalone/_index.md [hunk 2]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
 
 ## Why OCI
 
-Running Portworx as a runC container eliminates any cyclical dependencies between a Docker container consuming storage from the Portworx container. It also enables you to run your Linux containers without a Docker daemon completely, while still getting all of the advantages of a Linux container and cloud native storage from Portworx.
+Running Portworx as a runC container eliminates any cyclical dependencies between the Portworx container the Docker container consuming storage from the Portworx container. It also enables you to run your Linux containers without a Docker daemon completely, while still getting all of the advantages of a Linux container and a cloud-native storage solution provided by Portworx.
 
 ## Install
 
-### Prerequisites
+To install and setup the Portworx OCI bundle, you will follow a 3-step process:
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]: I think you can eliminate the list of steps unless they're long or complex. Up to you. 

```suggestion
To install and setup the Portworx OCI bundle, perform the following steps:
```

**Comment by andreipope** [comment]
> @andreipope Good stuff. Your grammar is much improved in these last PRs.
> 
> I had some smaller comments.

@nathan-px Thanks! Initially, the grammar wasn't good, but I was more focused on fixing it.

**Comment by andreipope** [comment]
@vinayakshnd Kind reminder.

File: content/install-with-other/docker/standalone/_index.md [hunk 3]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
 
 ## Why OCI
 
-Running Portworx as a runC container eliminates any cyclical dependencies between a Docker container consuming storage from the Portworx container. It also enables you to run your Linux containers without a Docker daemon completely, while still getting all of the advantages of a Linux container and cloud native storage from Portworx.
+Running Portworx as a runC container eliminates any cyclical dependencies between the Portworx container the Docker container consuming storage from the Portworx container. It also enables you to run your Linux containers without a Docker daemon completely, while still getting all of the advantages of a Linux container and a cloud-native storage solution provided by Portworx.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]: > Running Portworx as a runC container eliminates any cyclical dependencies between the Portworx container the Docker container consuming storage from the Portworx container.

You have two containers listed here, I think one was meant to be deleted.

File: content/install-with-other/docker/standalone/_index.md [hunk 4]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
...
+2. Configure the Portworx OCI
+3. Enable and start the Portworx service
 
-The installation and setup of PX OCI bundle is a 3-step process:
+### Prerequisites
 
-1. Install PX OCI bits
-2. Configure PX OCI
-3. Enable and start Portworx service
+* _SYSTEMD_: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system \(that is the `systemctl` command works\).
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]: What are these: `_SYSTEMD_:`? If they're just bullet list intros, I think we've started standardizing on bolding them with no special casing. Something like: 
```suggestion
* **Systemd**: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system and the `systemctl` command works.
```
```suggestion
* _SYSTEMD_: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system \(that is the `systemctl` command works\).
```

File: content/install-with-other/docker/standalone/_index.md [hunk 5]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
...
+  * If you are running Ubuntu 16.04, CentOS 7 or CoreOS v94 \(or newer\), `systemd` is already installed and no actions are required.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]:
```suggestion
  * If you are running Ubuntu 16.04, CentOS 7 or CoreOS v94 or newer, `systemd` is already installed and no actions are required.
```

File: content/install-with-other/docker/standalone/_index.md [hunk 6]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
...
+* _SCHEDULERS_: If you are installing Portworx into a **Kubernetes** or **Mesosphere DC/OS** cluster, we recommend to use [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package. It provides tighter integration, and better overall user experience.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]:
```suggestion
* _SCHEDULERS_: If you are installing Portworx into a **Kubernetes** or **Mesosphere DC/OS** cluster, Portworx recommends using [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package.
```

File: content/install-with-other/docker/standalone/_index.md [hunk 7]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
...
+* _FIREWALL_: Ensure ports 9001-9022 are open between the cluster nodes that will run Portworx.
+* _NTP_: Ensure all nodes running Portworx are time-synchronized. The NTP service should be configured and running.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]:
```suggestion
* _NTP_: Ensure all nodes running Portworx are time-synchronized by configuring and running the NTP service.
```

File: content/install-with-other/docker/standalone/_index.md [hunk 8]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
...
+* _KVDB_: Portworx requires a key-value database like etcd or consul. A highly available etcd cluster with persistent storage is preferred. See the [etcd installation](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/etcd) page for more details.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]:
```suggestion
* _KVDB_: Portworx requires a key-value database like etcd or consul; Portworx recommends a highly available etcd cluster with persistent storage. See the [etcd installation](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/etcd) page for more details.
```

File: content/install-with-other/docker/standalone/_index.md [hunk 9]
```diff
@@ -9,43 +9,43 @@ series: px-docker-install
...
+* _STORAGE_: At least one of the Portwox nodes should have extra storage available, either as a unformatted partition or as a disk-drive.  Also, please note that the storage devices explicitly passed to `px-runc` as command line arguments \(for example `px-runc ... -s /dev/sdb -s /dev/sdc3`\) will be automatically formatted during the installation process.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]: Would it make sense to move the note in the latter half of this prerequisite into a warning before the command in the step itself?

```suggestion
* _STORAGE_: At least one of the Portwox nodes should have extra storage available, either as an unformatted partition or as a disk-drive. Note that the storage devices explicitly passed to `px-runc` as command-line arguments \(for example, `px-runc ... -s /dev/sdb -s /dev/sdc3`\) will be automatically formatted during the installation process.
```

File: content/install-with-other/docker/standalone/_index.md (hunk 3 above)
Comment (andreipope) [reply content/install-with-other/docker/standalone/_index.md]: @nathan-px Nice catch. Thanks!

File: content/install-with-other/docker/standalone/_index.md (hunk 4 above)
Comment (andreipope) [reply content/install-with-other/docker/standalone/_index.md]: @nathan-px That's probably meant as a quick way of checking that `systemd` is installed. 
What about the following?
*Systemd**: The installation procedure assumes that the [systemd](https://en.wikipedia.org/wiki/Systemd) package is installed on your system. You can check if `systemd` is installed by entering the following command:

```text
systemctl
```

Comment (nathan-px) [reply content/install-with-other/docker/standalone/_index.md]: Good idea, let's do it your way.

Comment (andreipope) [reply content/install-with-other/docker/standalone/_index.md]: @nathan-px Thanks. It's updated.

**Comment by andreipope** [comment]
@nathan-px @vinayakshnd Hey guys, please review when you have some time.

File: content/install-with-other/docker/standalone/_index.md [hunk 10]
```diff
@@ -9,43 +9,49 @@ series: px-docker-install
...
+    ```text
+    systemctl
+    ```
+  
+    If you are running Ubuntu 16.04, CentOS 7 or CoreOS v94 or newer, `systemd` is already installed, and no actions are required.
 
-1. Install PX OCI bits
-2. Configure PX OCI
-3. Enable and start Portworx service
+* **Schedulers** : If you are installing Portworx into a Kubernetes or Mesosphere DC/OS cluster, Portworx recommends using [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]: There's an extra space after the colon on some of these 

```suggestion
* **Schedulers**: If you are installing Portworx into a Kubernetes or Mesosphere DC/OS cluster, Portworx recommends using [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package.
```

**Comment by nathan-px** [comment]
It looks good to me other than the last comment.

**Comment by andreipope** [comment]
> It looks good to me other than the last comment.

@nathan-px Nice catch! It's updated now. Thanks.

**Comment by andreipope** [comment]
@vinayakshnd Can you please review this PR when you get a chance?

File: content/install-with-other/docker/standalone/_index.md [hunk 11]
```diff
@@ -9,43 +9,49 @@ series: px-docker-install
...
+    systemctl
+    ```
+  
+    If you are running Ubuntu 16.04, CentOS 7 or CoreOS v94 or newer, `systemd` is already installed, and no actions are required.
 
-1. Install PX OCI bits
-2. Configure PX OCI
-3. Enable and start Portworx service
+* **Schedulers**: If you are installing Portworx into a Kubernetes or Mesosphere DC/OS cluster, Portworx recommends using [Stork](https://github.com/libopenstorage/stork), the Portworx-specific scheduler package.
+* **Firewall** : Ensure ports 9001-9022 are open between the cluster nodes that runs Portworx.
```
Comment (nathan-px) [inline content/install-with-other/docker/standalone/_index.md]: @andreipope There are extra spaces after the colon on multiple of these.

**Comment by vilasd** [comment]
@zoxpx please review this PR

File: content/shared/install-with-other-docker-runc-configure-portworx.md [hunk 12]
```diff
@@ -27,20 +27,34 @@ sudo /opt/pwx/bin/px-runc install -c MY_CLUSTER_ID \
...
-#### Modifying the Portworx configuration
+**Adding a metadata drive:**
 
-After the initial installation, you can modify the Portworx configuration file at `/etc/pwx/config.json` (see [details](/shared/install-with-other-docker-config-json)) and restart Portworx using `systemctl restart portworx`.
+You can configure Portworx to use a metadata drive for storing the internal KVdb as well as internal Portworx metadata such as journal or timestamp data. Under normal circumstances, using a dedicated metadata drive improves the performance of your Portworx cluster. Note that the size of the metadata drive must be greater than 64Gi.
+
+To add a metadata drive, run `px-runc` with the `-metadata` flag and pass it the disk device name as in the following example:
+
+```text
+/opt/pwx/bin/px-runc install -name portworx -c MY_CLUSTER_ID -k etcd:http://70.0.5.221:2379 -b -s /dev/sdd -s /dev/sde -s /dev/sdf -v /mnt:/mnt:shared -metadata /dev/sdc -e PX_CH_DEFAULT=false -e DISABLE_AUTO_DIAGS=true -m ens33 -d ens33
```
Comment (zoxpx) [inline content/shared/install-with-other-docker-runc-configure-portworx.md]: Let's please update all our examples on this page to `-m eth1 -d eth2` (if -m/-d were used in the examples).

Note that `-m` changes Management Network interface, while `-d` changes the Data Network interface.
Now, for over 20 years the Network interfaces on Linux were in `eth[0-9]` format (e.g. eth0, eth1, eth2..); about 5-6 years ago the `enp#s#` notation appeared (I've first seen in w/ CentOS6), and honestly I can't even recognize `ens33` as a network interface.
So, IMO, it's best to use the old / most common notation -- most of our audience will be aware of it, and instantly recognize eth# notation as a "network interface"

File: content/install-with-other/docker/standalone/_index.md [hunk 13]
```diff
@@ -144,4 +152,6 @@ sudo /opt/pwx/bin/px-runc run -c MY_CLUSTER_ID \
 [ hit Ctrl-C ]
 ```
 
-{{<info>}}**Migrating from Portworx containers to Portworx OCI**: If you already had Portworx running as a Docker container (Portworx 1.2.10 and lower) and now want to upgrade to runC, follow the instructions at [Migrate Portworx installed using Docker to OCI/runc](/install-with-other/docker/standalone/migrate-docker-to-oci). {{</info>}}
+{{<info>}}
```
Comment (zoxpx) [inline content/install-with-other/docker/standalone/_index.md]: Let's please remove this "Interactive/Foreground mode" paragraph.
At first I though it wold be useful, but turns out nobody used it  (or was aware of it) -- so I think it's more harm (i.e. confusion) than it's useful.

File: content/shared/install-with-other-docker-shared-runc-install-bundle.md [hunk 14]
```diff
@@ -23,4 +22,7 @@ sudo docker run --entrypoint /runc-entry-point.sh \
     $latest_stable
 ```
 
-{{<info>}}Running the Portworx OCI bundle does not require Docker, but Docker will still be required to _install_ the Portworx OCI bundle.  If you do not have Docker installed on your target hosts, you can download this Docker package and extract it to a root tar ball and manually install the OCI bundle.{{</info>}}
+{{<info>}}Your target hosts do not require Docker to run the Portworx OCI bundle, but they do require Docker to install it. If you have not installed Docker on your target hosts, you can install the OCI bundle by doing the following:
```
Comment (zoxpx) [inline content/shared/install-with-other-docker-shared-runc-install-bundle.md]: I'd recommend to remove this paragraph  (also the 1./2. points below).  It does not provide useful information, and it's confusing.
We could add the actual details how to create and install from the tarballs -- but honestly we used this (convoluted) procedure at customer's only once in 3 years, so I don't see any hard in dropping the subject and not mentioning it at all.

**Comment by nathan-px** [comment]
captured Z's other feedback to be implemented later when bandwidth permits: https://portworx.atlassian.net/browse/PD-377

**Pull request closed without merging**